robber/helper.py

```diff
@@ -0,0 +1,27 @@
+class Helper:
+    @staticmethod
+    def build_expected_params_string(expected, args, kwargs):
+        if not expected:
+            expected_params_str = 'no arguments'
```

Comment:
This should be returned soon to omit the `else` part and make it easier to read.